src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -76,3 +82,27 @@ def load(self) -> list[Rule]:
         except NotFound:
             msg = "Please run: databricks labs ucx table-mapping"
             raise ValueError(msg) from None
+
+    def skip_table(self, backend: StatementExecutionBackend, schema: str, table: str):
+        # Marks a table to be skipped in the migration process
+        try:
+            backend.execute(f"ALTER TABLE `{schema}`.`{table}` SET TBLPROPERTIES('{self.UCX_SKIP_PROPERTY}' = true)")
+        except NotFound as nf:
+            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(nf):
+                logger.error(f"Failed to apply skip marker for Table {schema}.{table}. Table not found.")
+            else:
+                logger.error(nf)
```

Comment:
Why do you need to catch exceptions just to log them?..

Reply:
To prevent the nasty log trace.